pennylane/interfaces/tensorflow.py

```diff
@@ -338,6 +333,14 @@ def execute(tapes, device, execute_fn, gradient_fn, gradient_kwargs, _n=1, max_d
     parameters = []
     params_unwrapped = []
 
+    if isinstance(device, qml.devices.experimental.Device):  # pragma: no-cover
+        # assumes all tapes have the same shot vector
+        has_partitioned_shots = tapes[0].shots.has_partitioned_shots
+        vjp_shots = legacy_shots = None
+    else:

```

Comment:
What will `vjp_shots` be in the `else` case? Or will it not be defined?

Reply:
sorry, I used the less-common `x = y = value` Python syntax, because both things are equal to `device.shot_vector`